data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,174 @@
+	on offer
+		"salary: Acorn Delights Dividends" = 500
+		payment 30000
+		log `Helped the famous Hai chef Eeeya find her granddaughter, a trauma surgeon named Anoo. Both joined the Unfettered hoping they can, "set them on a more productive path." Received shares of Acorn Delights stock which pay 500 credits a day in dividends.`
+		conversation
+			`You're ushered quickly from the landing platform to a nearby hospital. Unfettered Hai nearby are muttering in their language, but you keep hearing the Hai word for, "cookies" as they eye Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. It seems the Unfettered value function and progress, rather than aesthetics and comfort like their southern brethren.`
+			`	As you rise through the main tower of the hospital, you're surprised at how advanced it is. Despite their willingness to die in battle, the Unfettered seem to value the lives of the survivors. Perhaps this is to heal them quickly to fight again?`
+			`	You reach a high floor, filled with personal offices. Eeeya takes you to a door at the end of a long hallway.`
+			choice
+				`	(Open the door.)`
+			`	 After the door opens, Eeeya and Anoo run to each other's arms. A few minutes later, they begin to speak. Eventually the conversation turns to Anoo's new occupation.`
+			`	"Why did you come here, Anoo? The Unfettered hate us."`
+			`	"They don't hate US, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our race is fading into history through a slow death of mediocrity. Most Unfettered see warfare as our only salvation.`
+			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
+			`	"If enough good people join the Unfettered, we can change their minds; set them on a more productive path. They'll need good doctors."`
+			`	Eeeya smiles at her granddaughter, "They'll need a good cook too."`
+			choice
+				`	"I'm glad to have reunited you."`
+				`	"I'd like to be paid now."`
+			`	Eeeya turns to you and says, "I've found my new home, Captain <last>. I suppose payment is in order. I'm the owner of Eeeya's Acorn Delights, a luxury baked goods company based on Allhome.`
+			`	"I only have 30,000 credits with me, but I'll transfer some Acorn Delights stocks to your name. Your quarterly dividends will be divided into daily payments of 500 credits."`
+			`	Before you leave, Eeeya unpacks a feast of cookies for Anoo and her patients. Their aromatic fragrance quickly overpowers the wounds and suffering from war. Every cookie has perfect imperfections of crackled texture over the chewy center which melts into your mouth. If food could win wars, this would be the Unfettered's secret weapon.`
```

Comment:
This makes me want cookies. Nice description. `perfect imperfections` is a great choice.